docs/releases/adding-new-firmwares.md

```diff
@@ -0,0 +1,25 @@
+# Adding New Firmwares
+
+In case we are about to release both Suite and firmwares we want to add the signed firmwares during the Freeze so QA has the whole thing to test.
+
+## Add firmwares
+
+1. Complete the firmware release process including firmware signing.
+2. Add firmwares to [webwallet-data](github.com/trezor/webwallet-data/) and modify its `releases.json` file. See e.g. [a1831647](https://github.com/trezor/webwallet-data/commit/f8ed15a8999689e7692b8fc4c00b7aaef25d8011) for an example.
+3. Deploy them to data.trezor.io. _This is currently done manually and should be automated._
+4. Modify `releases.json` in Connect. See [5350f7ee](https://github.com/trezor/connect/commit/5350f7eef30a2137127636e8d8c42238ddc0d14c) for example.
+
+## Publish Connect to NPM and bump in Suite
+
+5. Publish Connect as beta. See [Connect: Updating NPM to beta](https://github.com/trezor/connect/blob/db7139e2cd22dd0d9ea9dc3f0725dbd13a57f691/docs/deployment/index.md#beta) on how to do that.
```

Comment:
```suggestion
5. Publish Connect as beta. See [Connect: Updating NPM to beta](https://github.com/trezor/connect/blob/develop/docs/deployment/index.md#beta) on how to do that.
```

Reply:
The hash is there intentionally. This way the content stays and it is guaranteed to will not lead somewhere else or to 404. As it might happen with `develop` if someone in future modifies or deletes the content.

Reply:
ah, ok!